test/endtoend/create-react-app/README.md

```diff
@@ -1034,12 +1074,12 @@ using `fetch()` to access it.
 
 ### Node
 Check out [this tutorial](https://www.fullstackreact.com/articles/using-create-react-app-with-a-server/).
-You can find the companion GitHub repository [here](https://github.com/fullstackreact/food-lookup-demo).
+You can find the Companion GitHub repository [here](https://github.com/fullstackreact/food-lookup-demo).
 
 ### Ruby on Rails
 
 Check out [this tutorial](https://www.fullstackreact.com/articles/how-to-get-create-react-app-to-work-with-your-rails-api/).
-You can find the companion GitHub repository [here](https://github.com/fullstackreact/food-lookup-demo-rails).
+You can find the Companion GitHub repository [here](https://github.com/fullstackreact/food-lookup-demo-rails).
```

Comment:
The companions in this file don't refer to Companion :smile: We might as well nuke this readme, it's just here because create-react-app made it

Reply:
Haha, oops! Cool, done!